packages/vscode.vm/vscode.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>vscode.vm</id>
-    <version>1.85.2.20240507</version>
+    <version>1.85.2.20240515</version>
```

Comment:
https://github.com/mandiant/VM-Packages/issues/972

Reply:
Fixed in https://github.com/mandiant/VM-Packages/pull/1051, closing this PR so that we can fix this version mitmatch before updating the package.